packages/terra-demographics-banner/src/_utils.jsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
```

Comment:
Usually, for the shared component, we can create a private component file to contain it. 
But dues to the array of React elements, `personDetails applicationIdentifiers` have to be the method. To avoid that, I may need to add a div to wrap it. But I'm not sure we want to that.

The reason to use `utils` is in general, these two methods are reused by other code.